.reuse/dep5

```diff
@@ -64,22 +64,10 @@ Files: examples/*
 Copyright: 2017 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
 License: Apache-2.0
 
-Files: integrations/completions/*
-Copyright: 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
-License: Apache-2.0
-
-Files: integrations/jenkins/*
-Copyright: 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
-License: Apache-2.0
-
-Files: integrations/schemas/*
+Files: integrations/*
 Copyright: 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
 License: Apache-2.0
 
-Files: integrations/tekton/*
```

Comment:
The Google attribution is already in the NOTICE file but this is the machine readable version so I would not remove it.